one_fm/one_fm/payroll_utils.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# encoding: utf-8
+from __future__ import unicode_literals
+from frappe import _
+import frappe
+from frappe.utils.user import get_user_fullname
+from one_fm.api.notification import create_notification_log
+from frappe.utils.user import get_users_with_role
+from frappe.utils import (
+    getdate,
+    today,
+    get_url
+)
+
+@frappe.whitelist()
+def get_wage_for_employee_incentive(employee, rewarded_by, on_date=today()):
+    '''
+        this function returns the wage of an employee based on the rewarded_by value
+        if rewarded_by == "Number of Daily Wage" returns basic_salary/30
+        if rewarded_by == "Percentage of Monthly Wage" returns base(total salary with all allowances) noted in the salary strucure assignment
+
+        args:
+            employee: employee ID
```

Comment:
set an example for employee ID like (eg: HR-EMP-00001) or (eg: 1511004IN188) to define the employee id clearly if it is **ERP id** or **company employee ID**